src/storyblok-auth-api/auth-handler.ts

```diff
@@ -17,11 +18,23 @@ export const authHandler = (
       res.writeHead(400).end()
       return
     }
+
+    const internalAdapter = createInternalAdapter({
```

Comment:
Creating the plugin here with the default `cookieAdapter` instead of passing it into `authHandler`, as this feature is not customisable and therefore should stay hidden.